tests/regression/README.md

```diff
@@ -3,21 +3,29 @@ owasp-crs-regressions
 
 Introduction
 ============
-Welcome to the OWASP Core Rule Set regression testing suite. This suite is meant to test specific rules in OWASP CRS version 3. The suite is designed to uses preconfigured IDs that are specific to this version of CRS. The tests themselves can be run without CRS and one would expect the same elements to be blocked, however one must override the default Output parameter in the tests.
+Welcome to the OWASP Core Rule Set regression testing suite. This suite is meant to test specific rules in OWASP CRS version 3. The suite is designed to use pre-configured IDs that are specific to this version of CRS. The tests themselves can be run without CRS and one would expect the same elements to be blocked, however one must override the default Output parameter in the tests.
 
 Installation
 ============
-The OWASP Core Rule Set project was part of the effort to develop FTW, the Framework for Testing WAFs. As a result, we use this project in order to run our regression testing. FTW is designed to use existing Python testing frameworks to allow for easy to read web based testing, provided in YAML. You can install FTW by from the repository (at https://github.com/CRS-support/ftw) or by running pip.
+The OWASP Core Rule Set project was part of the effort to develop FTW, the Framework for Testing WAFs. As a result, we use this project to run our regression testing. FTW uses existing testing frameworks to perform human-friendly web-based testing, provided in YAML. You can install FTW with Go or Python:
+ 
+```sh
+# Go: https://github.com/coreruleset/go-ftw#install
+go install github.com/coreruleset/go-ftw@latest
 
-```pip install -r requirements.txt```
+# Python: https://github.com/coreruleset/ftw#installation
+git clone https://github.com/coreruleset/ftw.git
+cd ftw
+pip install -r requirements.txt 
+```
 
 This will install FTW as a library. It can also be run natively, see the FTW documentation for more detail.
 
 Requirements
 ============
-There are Three requirements for running the OWASP CRS regressions.
+There are three requirements for running the OWASP CRS regressions.
 
-1. You must have ModSecurity specify the location of your error.log, this is done in the config.ini file. If you are using nginx you can use the parameter --config=modsec3-nginx (as specified in config.ini)
+1. You must have ModSecurity specify the location of your error.log, this is done in the config.ini file. If you are using nginx you can use the parameter `--config=modsec3-nginx` (as specified in config.ini)
```

Comment:
This is not true anymore. Was for the python version only.

Reply:
How about changes like this:

---

There are three requirements for running the OWASP CRS regressions:

1. Create `.ftw.yaml` for your environment. (see Section [Yaml Config File](https://github.com/coreruleset/go-ftw#yaml-config-file) in go-ftw for more details)
2. Specify your error.log location from ModSecurity in `.ftw.yaml`.
3. Make sure ModSecurity is in `DetectionOnly` (or anomaly scoring) mode.
4. Disable IP blocking based on previous events.

To accomplish 3 and 4, you may use the following rule in your setup.conf:

```
SecAction "id:900005,\
  phase:1,\
  nolog,\
  pass,\
  ctl:ruleEngine=DetectionOnly,\
  ctl:ruleRemoveById=910000,\
  setvar:tx.blocking_paranoia_level=4,\
  setvar:tx.crs_validate_utf8_encoding=1,\
  setvar:tx.arg_name_length=100,\
  setvar:tx.arg_length=400,\
  setvar:tx.max_file_size=64100,\
  setvar:tx.combined_file_sizes=65535"
```

> Note: The test suite compares **timezones** -- if your test machine and your host 
> machine are in different timezones which can cause bad results

Reply:
Looks better. Right now we don't compare times anymore in logs, we [use a marker](https://github.com/coreruleset/go-ftw#how-log-parsing-works) to find the right request. So the note doesn't apply. You also mention _three_ requirements, but four are listed. Also, if we are in _DetectionOnly_ mode, you won't be blocked in step 4.

Reply:
I see. Just to make sure, there are only three modes that can be configured via `MODSEC_RULE_ENGINE` currently: On / Off / DetectionOnly.

Reply:
Yes, `On | Off | DetectionOnly` are the modes.

Reply:
Yes. And I think adding `secAction` is not necessary either. updated in [#68ad8de](https://github.com/coreruleset/coreruleset/pull/3232/commits/68ad8de395625ef09007edb4df73759b026908ab)